Gems/Atom/RPI/Code/Include/Atom/RPI.Public/WindowContext.h

```diff
@@ -92,6 +92,9 @@ namespace AZ
             // The handle of the window that this is context describes
             AzFramework::NativeWindowHandle m_windowHandle;
 
+            // TODO comment
```

Comment:
Seems like this comment can be filled out now.
Would be better to use a WSISurface::Descriptor though?